edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -1,9 +1,11 @@
 // Copyright (c) Microsoft. All rights reserved.
 
 use std::collections::HashMap;
-use std::convert::From;
+use std::convert::{From, TryInto};
```

Comment:
Just remove `From`. It's already in prelude so it shouldn't have been explicitly `use`d anyway.